demo/menu-bar-items-demos.html

```diff
@@ -0,0 +1,138 @@
+<dom-module id="menu-bar-items-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Disabled Items</h3>
+    <p>
+      Both menubar and submenu items can be marked as disabled by setting the <code>disabled</code> property on
+      the item object. Disabled items are excluded from the keyboard navigation. Clicking disabled item does not
+      select it, and neither closes nor opens submenu.
+    </p>
+    <vaadin-demo-snippet id="menu-bar-items-demos-disabled" when-defined="vaadin-menu-bar">
+      <template preserve-content>
+        <vaadin-menu-bar></vaadin-menu-bar>
+        <p>Selected item: <i></i></p>
+        <script>
+          window.addDemoReadyListener('#menu-bar-items-demos-disabled', function(document) {
+            const menu = document.querySelector('vaadin-menu-bar');
+            menu.addEventListener('item-selected', function(e) {
+              document.querySelector('i').textContent = JSON.stringify(e.detail.value);
+            });
+            menu.items = [
+              {
+                text: 'Project',
+                children: [{text: 'Manage'}, {text: 'Delete', disabled: true}]
+              },
+              {text: 'Permissions', disabled: true},
+              {text: 'Help'}
+            ];
+          });
+        </script>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Checkable Items</h3>
+    <p>
+      Submenu items can be marked as selected using the <code>checkable</code> and <code>checked</code> properties.
```

Comment:
Note that `checkable` isn't in the items API, but it's just a name that's used in the demo. `foobar` would work just as fine as a marker for items that the demo code marks as `cheked` on click

Reply:
Updated.

Reply:
I think `checkable` is still a better term to use compared to `canCheck`. So in my opinion renaming the property was unnecessary. The changed documentation makes it clear that this property is not part of the API.

Reply:
Maybe could be made even more clearer by stating explicitly that "...is just used for the demo and not part of the items API"

Reply:
Done.